src/strawchemy/dto/types.py

```
@@ -185,11 +185,41 @@ def __post_init__(self) -> None:
         if self.exclude:
             self.include = "all"
 
+    @classmethod
+    def from_include(cls, include: IncludeFields | bool | None = None, purpose: Purpose = Purpose.READ) -> Self:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)